libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -300,6 +327,20 @@ - (void)replayRequestIfRequired:(NSData *)data response:(NSURLResponse *)respons
             [delegate request:request didFailLoadWithError:error];
         }
         [[SFRestAPI sharedInstance] removeActiveRequestObject:request];
+        if (!self.pendingRequestsBeingProcessed) {
```

Comment:
Send any pending requests once the initial refresh request has completed successfully.